analysis/pseudobulk-bulk-prediction/results/SCPCP000002_ORA-odds-ratios.tsv

```diff
@@ -0,0 +1,53 @@
+geneset_cell_type	modality	p_value	p_adj_bh	or_bulk	or_pseudobulk
```

Comment:
Why do we need both `or_bulk` and `or_pseudobulk` for every row here rather than `odds_ratio` and let `modality` tell you which one it is?

Reply:
hm, yes let me check this pivoting.

Reply:
Noting I updated this to make it long all around in https://github.com/AlexsLemonade/scpca-paper-figures/pull/175/commits/97bd09964fd19d344cbdc3151e4b301f6d1380f1, but plan to wait to re-render notebooks in case there are more reviews.